create_jenkins_job.py

```diff
@@ -172,7 +172,7 @@ def main(argv=None):
             job_name = job_name[0:-5]
         job_data['time_trigger_spec'] = '0 12 * * *'
         job_data['cmake_build_type'] = 'None'
-        job_data['ament_test_args_default'] = '--retest-until-fail 20'
+        job_data['ament_test_args_default'] = '--retest-until-fail 20 -LE lint'
```

Comment:
The label is `linter` (e.g. see https://github.com/ament/ament_lint/blob/27b56bf60ff1b7bc039cd8403b28373a50449a3f/ament_cmake_pep8/cmake/ament_pep8.cmake#L57).